issue_06/script.js

```diff
@@ -11,21 +11,27 @@ const contents = [{
 }];
 
 const ul = document.getElementById('js-ul');
-const checkContents = new Promise((resolve) => {
-    resolve(contents);
-});
-checkContents.then((value) => {
+const wait = (sec) => {
```

Comment:
IMO
I don't think this constant name is appropriate. This is because waiting is not the goal here.

Reply:
It is OK for me! I think the goal here is to get the data in the Promise function. So, ``getData`` or ``getList`` etc.. may be fine. It's just an idea, no need to modify it！